Px.Utils/PxFile/Data/DataValueParsers.cs

```diff
@@ -0,0 +1,281 @@
+﻿using PxUtils.Models.Data;
+using PxUtils.Models.Data.DataValue;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+
+namespace PxUtils.PxFile.Data
+{
+    public static class DataValueParsers
+    {
+        /// <summary>
+        /// This method uses a fast, but potentially unsafe, parser to convert the characters into a <see cref="DoubleDataValue"/>.
+        /// It is important that the input has been validated before using this method since the method does not perform any validation.
+        /// Invalid input will result in undefined behavior.
+        /// </summary>
+        /// <param name="buffer">The characters to parse.</param>
+        /// <param name="len">The number of characters to parse.</param>
+        /// <returns>A <see cref="DoubleDataValue"/> instance representing the parsed value.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static DoubleDataValue FastParseDoubleDataValueDangerous(char[] buffer, int len)
+        {
+            if (buffer[0] == '"')
+            {
+                if (buffer[1] == '-') return new DoubleDataValue(0, DataValueType.Nill);
+                return new DoubleDataValue(0, (DataValueType)(len - 2));
```

Comment:
I'm not fan of comments but this needs a comment, both here and in DataValueType

Reply:
Added comments.